README.md

```diff
@@ -1 +1,7 @@
-# hello-rails-back-end
\ No newline at end of file
+# Hello App
+
+> This includes the backend of the hello app which is built with rails 7.
+
+## Frontend
+
+> To checkout the frontend of this app. please refer to this [link](https://github.com/HammadSiddique/hello-react-frontend)
```

Comment:
- [ ] Please kindly update your `readme file` to reflect the changes for this milestone as this will make your `readme file` more informative and professional. Checkout this [template](https://github.com/microverseinc/readme-template) and adapt it to your project. Replace all the information in the various sections of the template with information pertaining to your project.